internal/service/alarms/internal/db/migrations/000003_create_alarm_event_record_table.up.sql

```diff
@@ -21,9 +21,8 @@ CREATE TABLE IF NOT EXISTS alarm_event_record (
     extensions JSONB, -- Additional data for extensibility
 
     -- O-RAN additional data to create AlarmEventNotification
-    resource_id UUID NOT NULL, -- Same as manager_cluster_id for caas alerts
-    resource_type_id UUID NOT NULL, -- Derived from manager_cluster_id
-    notification_event_type INT NOT NULL, -- Should be enum calculated from current alert
+    object_id UUID, -- Same as manager_cluster_id for caas alerts
```

Comment:
these are not nullable according to the spec and our api

Reply:
so this is there to simply capture all the events regardless if managedcluster id is present. the idea is that we capture these issue now and then let ACM folks know about it instead of dropping any alerts.